content/documents/ordercloud-supported-regions.mdx

```diff
@@ -0,0 +1,28 @@
+---
+type:           article
+title:          OrderCloud Supported Regions
+description:    Information about Order Cloud API servers for all supported regions and environments.
+author:         smitbmx
```

Comment:
In order for your image and name to show up on the article, you will need to follow these instructions in the README:

>  This is the author _identifier_, when set, Gatsby will query data in `root/src/data/author.json` to retrieve the author's name & title. Additionally, there should always be a `.jpg` image in `root/static/authors` for each author entry where the name of the image is the `author.id`. |